crates/phactory/src/system/mod.rs

```diff
@@ -455,8 +456,20 @@ pub struct System<Platform> {
     retired_versions: Vec<Condition>,
 }
 
-fn create_sidevm_service() -> Spawner {
-    let (service, spawner) = sidevm::service::service();
+thread_local! {
```

Comment:
It's temporarily used to configure the runtime thread poll in serde deserializing progress. We use `thread_local!` because there isn't a way to pass extra args to the Deserialize::deserialze.

Reply:
Please leave a comment

Reply:
Done